lib/buildFactory.js

```diff
@@ -168,7 +169,7 @@ class BuildFactory extends BaseFactory {
                     ]).then(([setup, teardown]) => {
                         modelConfig.steps = setup.concat(modelConfig.steps, teardown);
                         // Launcher is hardcoded to do some business in sd-setup-launcher
-                        modelConfig.steps.unshift({ name: 'sd-setup-launcher' });
+                        modelConfig.steps.unshift({ name: 'sd-setup-launcher', alwaysRun: true });
```

Comment:
And my thought on the first step alwaysRun flag is that I just want things to be standardized. So I want this step to have an `alwaysRun` key and I figure that the value makes more sense `true` than `false`. Not sure if that makes sense.